.pylintrc

```diff
@@ -80,7 +80,11 @@ disable=broad-except,
         too-many-arguments,
         too-many-return-statements,
         too-many-branches,
-        too-many-statements
+        too-many-statements,
+
+        missing-function-docstring,
+        missing-module-docstring,
+        missing-class-docstring
```

Comment:
Disabling linters? How come?

Reply:
Temporary. I would like to have other checks till I didn't write docstrings.

Reply:
Enabled this back.